packages/app/src/app/pages/Dashboard/Content/routes/PathedSandboxes/index.js

```diff
@@ -67,6 +65,9 @@ const PathedSandboxes = props => {
                 isLoading={loading}
                 possibleTemplates={possibleTemplates}
                 Header={<Navigation teamId={teamId} path={path} />}
+                Folders={
```

Comment:
This could be renamed to something more generic like `SubHeader`. This way we can maybe use it in the future for the `Recent Sandboxes` or `Search` page as well.

Reply:
Changed :)